lib/jaeger/client.rb

```diff
@@ -29,15 +31,23 @@ def self.build(host: '127.0.0.1',
                    flush_interval: DEFAULT_FLUSH_INTERVAL,
                    sampler: Samplers::Const.new(true),
                    logger: Logger.new(STDOUT),
-                   sender: nil)
+                   sender: nil,
+                   propagate_b3: false)
```

Comment:
Can you change this to `propagation_codec: PropagationCodec::JaegerCodec`? This allows users to pass `propagation_codec: Jaeger::Client::PropagationCodec::B3Codec`. The end behavior is the same but this allows extensions. E.g. We can later add [trace-context](https://github.com/w3c/trace-context) propagation by just adding a new module/class. In the current case we would need a new flag and it can get confusing. It also allows users to write their own propagation codec modules if they want and just inject those.